File: Examples/DotNET/SDK/CSharp/Images/UpdateJpgPropertiesWithoutStorage.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Imaging. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////

using System;
using Aspose.Cloud;

namespace Aspose.Imaging.Cloud.Examples.Images
{
    class UpdateJpgPropertiesWithoutStorage
    {
        static void Main()
        {
            string dataDir = Common.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
            string input = "sample1.jpg";
            string output = "output.jpg";
            
            string outPath = dataDir+output;

            Common.GetImagingSdk().Jpg.UpdateParametersOfJpgImage(outPath, dataDir + input);
        }
    }
}

File: Examples/DotNET/SDK/VisualBasic/Images/UpdateJpgPropertiesWithoutStorage.vb
'////////////////////////////////////////////////////////////////////////
' Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
'
' This file is part of Aspose.Imaging. The source code in this file
' is only intended as a supplement to the documentation, and is provided
' "as is", without warranty of any kind, either expressed or implied.
'////////////////////////////////////////////////////////////////////////

Imports System
Imports Aspose.Cloud

Namespace Aspose.Imaging.Cloud.Examples.Images
	Friend Class UpdateJpgPropertiesWithoutStorage
		Shared Sub Main()
			Dim dataDir As String = Common.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
			Dim input As String = "sample1.jpg"
			Dim output As String = "output.jpg"

			Dim outPath As String = dataDir & output

			Common.GetImagingSdk().Jpg.UpdateParametersOfJpgImage(outPath, dataDir & input)
		End Sub
	End Class
End Namespace
